src/ore/Cargo.toml

```diff
@@ -63,6 +63,7 @@ yansi = { version = "0.5.1", optional = true }
 [dev-dependencies]
 anyhow = { version = "1.0.66" }
 scopeguard = "1.1.0"
+maplit = "1.0.2"
```

Comment:
Vestigial from past iteration of the PR, I imagine!

Reply:
Oh yes, will make sure this gets reverted